app/src/main/java/ch/epfl/favo/auth/SignInActivity.java

```diff
@@ -48,7 +48,7 @@ protected void onCreate(Bundle savedInstanceState) {
     startActivityForResult(createSignInIntent(), RC_SIGN_IN);
   }
 
-  private void checkPlayServices () {
+  private void checkPlayServices() {
```

Comment:
Just as a suggestion, try not to include changes like this in a PR if this file is not relevant to the PR. A better solution would be to make another PR and call it something like "Cleaning up some code". It would make the lives of people reviewing this PR a lot easier.

Reply:
You're right but this should not happen if everyone formats his own code before pushing. Agree we should make another PR for cleaning and reformatting.